packages/react-native/.npmignore

```diff
@@ -25,6 +25,11 @@ android/.idea/scopes/scope_settings.xml
 android/.idea/vcs.xml
 android/*.iml
 
+# Expo config plugin
+plugin/src/
+plugin/__tests__/
+plugin/tsconfig.json
+
```

Comment:
Out of curiosity, why ignore the plugin source and tsconfig? They're pretty small and it seems like it removes the ability for consumers to patch things locally if needed, for no real benefit other than small space savings? Having the source is the sort of thing that isn't important to anyone until it really is all of the sudden

Reply:
I searched a bunch in the react-native-firebase repos and saw it was being excluded there, so added it here.

https://github.com/invertase/react-native-firebase/blob/master/packages/crashlytics/.npmignore#L31

I am impartial - but don't see the harm in leaving the src in the release, just tried to go for consistency.